src/annotator/anchoring/pdf.js

```diff
@@ -426,10 +458,10 @@ export function anchor(root, selectors) {
         const { pageIndex, anchor } = quotePositionCache[quote.exact][
           position.start
         ];
-        return anchorByPosition(pageIndex, anchor);
+        return anchorByPosition(pageIndex, anchor.start, anchor.end);
       }
 
-      return prioritizePages(position).then(pageIndices => {
+      return prioritizePages(position?.start ?? 0).then(pageIndices => {
         return findInPages(pageIndices, quote, position);
       });
```

Comment:
This handles the case were we try to anchor an annotation with a quote selector but no position selector. This possibility seemed to have been overlooked before.